numpy/fft/_pocketfft.py

```diff
@@ -113,10 +128,15 @@ def fft(a, n=None, axis=-1, norm=None):
     axis : int, optional
         Axis over which to compute the FFT.  If not given, the last axis is
         used.
-    norm : {None, "ortho"}, optional
+    norm : {"backward", "ortho", "forward"}, optional
```

Comment:
one question I have is whether the option `norm=None` should also be listed here or not; personally I think that it's better not to list it anymore, even thought it's accepted internally, because it would confuse someone reading through the docs; 

a few lines later we list that `backward` works as an alias for `None`, so codes with `norm=None` will still work, but having both `None` and `backward` in the kwarg values list could confuse ppl and make it look like `None` accounts to no normalization, which is what we're actually trying to resolve by introducing `norm=backward`

Reply:
I think it's a fairly common pattern that `=None` means the default option. So, instead of mentioning `None` explicitly I would just say that `"backward"` is the default.

Reply:
ok sounds good; you think I should also remove it from line 135 below, where it's explicitly said that `backward` is a `None` alias?

Reply:
Yes, I would say remove it.

Reply:
ok will do